java-checks/src/main/java/org/sonar/java/symexecengine/ExecutionState.java

```diff
@@ -145,7 +122,7 @@ private void reportIssuesToTopState(Set<Tree> trees) {
   // FIXME : Hideous hack for closeable to get "Ignored" variables
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)